raiden_contracts/tests/fixtures/user_deposit.py

```diff
@@ -3,15 +3,22 @@
 from raiden_contracts.constants import CONTRACT_USER_DEPOSIT
 
 
+@pytest.fixture(scope='session')
+def user_deposit_whole_balance_limit(custom_token_total_supply):
+    return custom_token_total_supply // 100
+
+
 @pytest.fixture(scope='session')
 def uninitialized_user_deposit_contract(
         deploy_tester_contract,
         custom_token,
+        user_deposit_whole_balance_limit,
 ):
+    print(user_deposit_whole_balance_limit)
```

Comment:
Is this `print` here intentionally, or is it left over from debugging?

Reply:
Indeed it's a leftover from debugging. I'll remove this in this PR.